.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,52 +1,69 @@
+        Thank you for helping us identify bugs and issues in our products.
 
-        Going through bugs and issues takes up a lot of time, so please be so kind and take a few minutes to fill out all the areas to the best of your ability.
+        We kindly ask that you take the time to:
+          - Search through [existing issues](https://github.com/ContinuumIO/anaconda-issues/issues) to verify whether your bug or issue has already been reported.
+          - Fill in all of the fields below, as best you can.
+          - Review and comply with our [Anaconda Community Code of Conduct](https://www.anaconda.com/community-code-of-conduct/).
 
-        There will always be more issues than there is time to do them, and so we will need to selectively close issues that don't provide enough information, so we can focus our time on helping people like you who fill out the issue form completely. Thank you for your collaboration!
-
-        There are also already a lot of open issues, so please take 2 minutes and search through existing ones to see if what you are experiencing already exists
-
-        Thanks for helping Anaconda be amazing. We are nothing without people like you helping build a better community 💐!
-
-        This issue type is for issues with installing Anaconda or Miniconda.
-
-        Anaconda Community Code of Conduct: https://www.anaconda.com/community-code-of-conduct/
+        Again, thank you for your contribution and for helping Anaconda better serve its community of users. 🐍
 
   - type: checkboxes
     id: checks
     attributes:
       label: Checklist
-      description: Please confirm and check all the following options.
+      description: "Please confirm the following:"
       options:
-        - label: I added a descriptive title
+        - label: I added a descriptive title.
           required: true
-        - label: I searched for other issues and couldn't find a solution or duplication
+        - label: I searched through [existing issues](https://github.com/ContinuumIO/anaconda-issues/issues) and couldn't find a solution or duplicate issue.
           required: true
-        - label: I already searched in Google and didn't find any good information or help
+        - label: I searched on the web (e.g. Google) and didn't find any helpful information.
           required: true
-        - label: I looked at the [docs](https://docs.anaconda.com/) and didn't see anything to help
+        - label: I searched the [Anaconda documentation](https://docs.anaconda.com/) and didn't find any helpful information.
           required: true
+
+  - type: checkboxes
+    id: product
+    attributes:
+      label: Impacted product
+      description: Please indicate the product(s) for which you are reporting the bug or issue.
+      options:
+        - label: Anaconda Distribution (formerly Anaconda Individual Edition)
+          required: false
+        - label: Miniconda
+          required: false
+        - label: A conda package
```

Comment:
I think this should be An Anaconda-built conda package  - we don't want issue reports for packages built by conda-forge, etc.

Reply:
Done. Thank you!